planetiler-core/src/main/java/com/onthegomap/planetiler/util/Wikidata.java

```diff
@@ -110,12 +112,14 @@ private static LongObjectMap<Map<String, String>> parseResults(InputStream resul
    *
    * @throws UncheckedIOException if an error occurs
    */
-  public static void fetch(OsmInputFile infile, Path outfile, PlanetilerConfig config, Profile profile, Stats stats) {
+  public static void fetch(OsmInputFile infile, Path outfile, PlanetilerConfig config, Profile profile, Stats stats,
+    Duration maxAge, int updateLimit) {
+
```

Comment:
Can you also add a `fetch` method too that sets the new arguments to whatever they need to be set to to retain the old behavior?

Reply:
Adjusted.